Automattic/jetpack
# [RNMobile] Update embed loading screen

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/5654

## Proposed changes:

* Updates the styling of the embed loading view 
* Renders the loading screen while the preview is being prepared


### Screens

**Still**
<img src="https://user-images.githubusercontent.com/744755/233724052-12aa180e-5f81-4883-a9d4-f752715e31f9.png" width="350" />

**Recording**


https://user-images.githubusercontent.com/744755/234397494-8547d636-60b1-4368-a25a-1c0681374d1e.mov






### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Add a VideoPress block and upload a video with a tall aspect ratio
* Verify that the loading screen appears the same as the embed player "processing" screen
* Wait for the video to be processed
* Notice the loading screen is rendered again at the default height ( ~200px)
* Notice the player height expand when the video loads

Note: VideoPress sometimes responds with the correct height for the video preview. In these cases the loading screen is not displayed after the video is processed.
